nav2_behavior_tree/test/CMakeLists.txt

```diff
@@ -5,3 +5,8 @@ add_subdirectory(plugins/condition)
 add_subdirectory(plugins/decorator)
 add_subdirectory(plugins/control)
 add_subdirectory(plugins/action)
+
```

Comment:
Would not including the directories here be better before adding the subdirectories?